docs/programming-oak.md

```diff
@@ -478,8 +481,8 @@ This has a little bit more boilerplate than testing a method directly:
 
 - The inbound gRPC channel that requests are delivered over has to be explicitly
   set up (`oak_tests::grpc_channel_setup_default`)
-- A separate thread running the Node's `oak_main` entrypoint is started
-  (`oak_tests::start_node`).
+- A separate thread running the Node's main entrypoint (assumed to be called
+  `oak_main`) is started (`oak_tests::start_node`).
```

Comment:
Can we remove the "assumed" part?

Reply:
Not really; for the simple `oak_tests::start_node()` helper, the entrypoint is assumed to be `oak_main`, as generated by `derive(OakExports)`.  Added a clarifying clause.